src/uct/api/tl.h

```diff
@@ -226,6 +226,8 @@ typedef ucs_status_t (*uct_ep_disconnect_func_t)(uct_ep_h ep, unsigned flags);
 
 typedef ucs_status_t (*uct_cm_ep_conn_notify_func_t)(uct_ep_h ep);
 
+typedef ucs_status_t (*uct_ep_query_func_t)(uct_ep_h ep, uct_ep_attr_t *ep_attr);
```

Comment:
move to internal header file uct_iface.h

Reply:
@yosefe Done